src/Application/Game.cpp

```diff
@@ -149,15 +147,6 @@ int Game::run() {
     }
     gameWindowHandler->removeFsmEventHandler(fsm.get());
 
-    //ShowMM7IntroVideo_and_LoadingScreen();
-
-    dword_6BE364_game_settings_1 |= GAME_SETTINGS_4000;
-
-    GUIWindow_MainMenu::drawCopyrightAndInit([&] {
```

Comment:
I have another branch which is going to remove these methods from the `GUIWindow_MainMenu` entirely but if you prefer I can grab those changes here as well.

Reply:
Let's do bite-sized PRs